### DISCUSSION TITLE
Important error

### DISCUSSION THREAD

**Discussion by anshuld1995** [community/community#19839] [Actions] [answered]
I'm getting a error i'am getting a error in importing xgboost it's shows  no model name xgboost

**Comment by Zi-Tao** [comment, marked as answer]
@anshuld1995 
https://bobbyhadz.com/blog/python-no-module-named-xgboost
https://stackoverflow.com/questions/40747738/importerror-no-module-named-xgboost

**Reply by anshuld1995** [reply]
thanks its work .

**Reply by anshuld1995** [reply]
thanks its working now

**Comment by Alvao-1** [comment]
when we forget to install the xgboost module before importing it or install it in an incorrect environment. To solve the error, install the module by running the pip install xgboost command
https://infinityclinic.co.uk/treatment/dermal-fillers-for-tear-trough/
